rooms/views.py

```diff
@@ -1,3 +1,169 @@
-from django.shortcuts import render
+import sys, json
 
-# Create your views here.
+from django.http       import JsonResponse
+from django.views      import View
+from django.db.models  import Avg, Count, Q
+from django.core.cache import cache
+
+from functools import reduce
+
+from .models import ReviewEvaluation, Room, RoomOption, Convenience, RoomType, RoomReview, Wish, Evaluation
+
+# AWS S3
+FILE_SERVER_PATH = 'https://groundbnb.s3.us-east-2.amazonaws.com/'
+
+# 소수 2째자리
+FOMATTER = lambda x: 0 if not x else float(format(x, '.2f'))
+
+class RoomFilterParam(View):
+    # 프론트에서 벡엔드로 날려야 되는 파라미터
+    def get(self, request):
+        room_options     = cache.get_or_set('room_options', {option.id : option.name for option in RoomOption.objects.all()})
+        room_convenience = cache.get_or_set('room_convenience', {option.id : option.name for option in Convenience.objects.all()})
+        
+        result = {
+            'room_type' : [
+                {
+                    'id'   : type.id,
+                    'name' : type.name 
+                } for type in RoomType.objects.all()
+            ],
+            'room_convenience' : [
+                {
+                    'id'   : key,
+                    'name' : value
+                } for key,value in room_convenience.items()
+            ],
+            'room_options' : [
+                {
+                    'id'   : key,
+                    'name' : value
+                } for key,value in room_options.items()
+            ]
+        }
+
+        return JsonResponse({'result_filter_param' : result}, status=200)
```

Comment:
여기는 한줄로 하는 것도 괜찮을 것 같은데요??

```suggestion
        result = {
            'room_type'        : [{'id' : type.id, 'name' : type.name } for type in RoomType.objects.all()],
            'room_options'     : [{'id' : key, 'name' : value} for key,value in room_options.items()],
            'room_convenience' : [{'id' : key, 'name' : value} for key,value in room_convenience.items()],
        }
```

Reply:
넵 훨씬 가독성이 좋아지네요~!
수정 완료했습니다.